Cron/DeleteBackups.php

```diff
@@ -0,0 +1,206 @@
+<?php
+declare(strict_types=1);
+
+namespace Overdose\CMSContent\Cron;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Exception\FileSystemException;
+use Magento\Framework\Stdlib\DateTime\DateTime;
+use Overdose\CMSContent\Model\Config\Cron\CronConfig;
+use Psr\Log\LoggerInterface;
+
+class DeleteBackups
+{
+    /**
+     * @var DirectoryList
+     */
+    private DirectoryList $directoryList;
+
+    /**
+     * @var DateTime
+     */
+    private DateTime $dateTime;
+
+    /**
+     * @var CronConfig
+     */
+    private CronConfig $cronConfig;
+
+    /**
+     * @var LoggerInterface
+     */
+    private LoggerInterface $logger;
+
+    /**
+     * @param DirectoryList $directoryList
+     * @param DateTime $dateTime
+     * @param CronConfig $cronConfig
+     * @param LoggerInterface $logger
+     */
+    public function __construct(
+        DirectoryList $directoryList,
+        DateTime $dateTime,
+        CronConfig $cronConfig,
+        LoggerInterface $logger
+    )
+    {
+        $this->directoryList = $directoryList;
+        $this->dateTime = $dateTime;
+        $this->cronConfig = $cronConfig;
+        $this->logger = $logger;
+    }
+
+    /**
+     * @return void
+     * @throws FileSystemException
+     */
+    public function execute()
+    {
+        if ($this->cronConfig->isCronEnabled()) {
```

Comment:
Better to do in a next way:
`
if (rule) {

 return null;

}

other code
`